Pages/HomePage/style.css

```
@@ -149,18 +158,61 @@ header :hover {
     text-decoration-line: underline;
 }
 
+
 footer {
     height: 73px;
     display: flex;
+    flex-direction: column;
     justify-content: space-between;
-    padding: 21px 31px;
-    background-color: var(--gray-main);
-    position: fixed;
-    bottom: 0;
+    height: 321px;
+    background-color: #0500ff;
+    padding: 31px 157px;
     width: 100%;
     box-shadow: 0px -4px 6px 0px #00000040;
 }
 
+.columns-container {
+    display: flex;
+    justify-content: space-between;
+    margin: 0 4rem;
+}
+
+.footer-column {
+    display: flex;
+    flex-direction: column;
+}
+
+.desktop-footer h3 {
+    color: var(--black);
+    font-size: 20px;
+    font-weight: 700;
+    margin-bottom: 20px;
+}
+
+.desktop-footer a {
+    color: var(--white);
+    text-decoration: none;
+    font-size: 16px;
+    font-weight: 500;
+    line-height: 24px;
+}
+
```

nas ancoras do footer coloca quando hover uma underline
```suggestion

.desktop-footer a : hover {
    text-decoration: underline;
}
```